scripts/install

```diff
@@ -109,10 +112,26 @@ __grpc_install_python_pkg() {
     CFLAGS=-I${brew_root}/include LDFLAGS=-L${brew_root}/lib pip install grpcio
 }
 
-# TODO: add __grpc_install_php_pkg
+__grpc_install_php_pkg() {
+    which 'phpize' >> /dev/null || {
+        echo "Could not detect phpize; Please make sure php5-dev is installed" 1>&2
+        return 1;
+    }
+    local brew_root=$(__grpc_brew_root) || return 1
+    # compile PHP extension from source (same release version as grpc)
+    # this should be replaced by 'pecl install grpc' at a later date
+    tar -C /tmp -zxvf `brew --cache grpc`
+    local grpc_src=`find /tmp -type d -name 'grpc-release*'`
+    cd $grpc_src/src/php/ext/grpc
+    phpize
+    ./configure --enable-grpc=$brew_root
+    make
+    sudo make install
```

Comment:
I am not sure about this. I probably shouldn't be doing `sudo` here right?
But I am not sure how to reliably get the compiled `grpc.so` extension file into the standard PHP extension directory which is usually under `/usr/lib/php5` folder.
Any advice?